### PR TITLE
Multiple Stops: GRC and Haigis

### LINKED ISSUE #6: Incorporate GRC and Haigis
Longer term enhancement, not necessary for initial rollout.

### PR DESCRIPTION
**Pull request by mboneil10**
Closes #6

### CODE REVIEW

File: spec/departure_comparator_spec.rb [hunk 1]
```diff
@@ -68,27 +68,11 @@
   describe 'compare' do
     context 'with scheduled routes' do
       it 'checks if every route is present and next departure has correct scheduled time' do
-        Bojangles.stub(:get_avail_departure_times!) do
```
Comment (mboneil10) [inline spec/departure_comparator_spec.rb]: Why does this still work with the deletion of the stub

Comment (mboneil10) [inline spec/departure_comparator_spec.rb]: With the deletion of the stub?

Comment (mboneil10) [reply spec/departure_comparator_spec.rb]: I'll rework these tests

Comment (Anbranin) [reply spec/departure_comparator_spec.rb]: :horse:

Comment (mboneil10) [reply spec/departure_comparator_spec.rb]: What does that mean? I'm confused.

Comment (Anbranin) [reply spec/departure_comparator_spec.rb]: Workhorse

Comment (mboneil10) [inline spec/departure_comparator_spec.rb]: ... I didn't even test the compare method! I don't know what I was thinking. Wow.

**Comment by mboneil10** [comment]
Messed up a few data structures in the mocks. Will fix and continue to build the spec file.

**Comment by mboneil10** [comment]
Oops need to run rubocop and gitignore...

**Comment by mboneil10** [comment]
Test coverage went down because I'm using mocks instead of calling methods directly. Think I'm done with testing multiple stops. Let me know @Anbranin

File: coverage/.last_run.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 81.54
+    "covered_percent": 77.12
```
Comment (Anbranin) [inline coverage/.last_run.json:3]: What's with the drop?

Comment (mboneil10) [reply coverage/.last_run.json:3]: I'm using mocks instead of calling methods from gtfs_parser that aren't related to multiple stops.

Comment (Anbranin) [reply coverage/.last_run.json:3]: Oops, didn't see your comment above sorry

**Review by Anbranin — APPROVED** [review]

**Pull request merged**